nilearn/regions/signal_extraction.py

```diff
@@ -151,6 +151,7 @@ def signals_to_img_labels(signals, labels_img, mask_img=None,
     --------
     nilearn.regions.img_to_signals_labels
     nilearn.regions.signals_to_img_maps
+    nilearn.input_data.NiftiMasker
```

Comment:
This function is more useable by users who wants to extract signals based on labels in atlases. I would put also ```nilearn.input_data.NiftiLabelsMasker : Signal extraction on labels images e.g. clusters```.

I would suggest to add also few details about what nilearn.input_data.NiftiLabelsMasker is ?
You can inspire from this reference:
https://github.com/nilearn/nilearn/blob/master/nilearn/plotting/img_plotting.py#L677

Reply:
You can skip my comment here. NiftiMasker is enough I think.